Makefile

```diff
@@ -7,16 +7,20 @@ SPHINXOPTS    ?=
 SPHINXBUILD   ?= sphinx-build
 SOURCEDIR     = .
 BUILDDIR      = _build
-VENV          = .sphinx/venv/bin/activate
+VENVDIR       = .sphinx/venv
+VENV          = $(VENVDIR)/bin/activate
 
 
 # Put it first so that "make" without argument is like "make help".
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-install:
+.PHONY: help
+
+
+$(VENVDIR):
```

Comment:
Will this detect changes to .sphinx/requirements.txt? If that file is changed, the installation must run again (at least when calling `make install`).